torch/_export/utils.py

```diff
@@ -42,6 +45,175 @@
 }
 
 
+def _collect_and_set_constant_attrs(
```

Comment:
nit: could you help add some context in the docstrings as to when/where these functions are used? e.g. in strict-mode, after dynamo export. Just makes it easier to parse through this file when a lot's going on.